src/Symfony/Bundle/FrameworkBundle/Templating/Helper/SessionHelper.php

```diff
@@ -72,26 +61,20 @@ public function hasFlash($name)
         return $this->getSession()->getFlashBag()->has($name);
```

Comment:
Request::getSession says `@return SessionInterface|null` so this is not save to be called.